sgv2-docsapi/src/main/java/io/stargate/sgv2/docsapi/service/schema/AuthorizedTableManager.java

```diff
@@ -15,8 +17,27 @@
 @Authorized
 public class AuthorizedTableManager extends TableManager {
 
+  /**
+   * {@inheritDoc}
+   *
+   * <p>Uses {@link
+   * io.stargate.sgv2.docsapi.service.schema.common.SchemaManager#getTableAuthorized(String, String,
+   * Function)}
+   */
   @Override
   protected Uni<Schema.CqlTable> getTable(String keyspaceName, String tableName) {
     return schemaManager.getTableAuthorized(keyspaceName, tableName, getMissingKeyspaceFailure());
   }
```

Comment:
nit: is the javadoc really needed? It just describes the implementation literally, if someone needs that information they could look at the method body.

Reply:
true, just a habit I have :)

Reply:
I don't have a strong opinion here but I think Javadocs are often exposed more prominently by IDEs (when javadoc artifact exists in Maven repo along with jar) than code; for code it may be necessary to click through.
Not a big deal but there is some value in javadocs beyond source code.